declarations_site/catalog/templatetags/catalog.py

```diff
@@ -17,6 +19,14 @@ def updated_querystring(request, params):
     return original_params.urlencode()
 
 
+@library.global_function
+@jinja2.contextfunction
```

Comment:
Do you really need both decorators? django_jina.library should cover both?

Reply:
If you look closely, you will observe that they have a pretty different set of parameters